### ISSUE TITLE
Groups that are able to post announcements can also see all announcements 

### ISSUE THREAD

**Issue by elhananjair** [nextcloud/announcementcenter#433]
### Steps to reproduce
1. Announce to a specific group let's say Group A
2. If a user is part of the announcing group but not Group A, he/she still can see the announcement

### Expected behavior
I don't know if this behavior is intentional but I was expecting that even if the user has a privilege to announce (Because the group he/she is in allowed to announce in admin settings), the announcement made to Group A (which the user is not a member of) shouldn't be seen by the user. 

### Actual behavior
This user can see all announcements made by a user who has permission to announce. 

### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

### Client configuration
**Browser:**

**Operating system:**

#### Nextcloud log (data/owncloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

**Comment by EmJotGeh** [comment]
I noticed that too and it shouldn't be like that.

**Comment by ffischer-lyrtron** [comment]
Same here, I notice the same...

**Comment by nickvergessen** [comment]
yeah for now this is intended and https://github.com/nextcloud/announcementcenter?tab=readme-ov-file#%EF%B8%8F-state-of-maintenance applies
